src/views/FAQPage/FAQItem.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <Fragment>
```

Comment:
instead of using a fragment, since this will be changing in the nearish future, we should favor a div wrapper in place of the fragment tag.